test/common/markdown/markdown.tests.ps1

```diff
@@ -32,16 +32,13 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
                 throw "node not found"
             }
         }
-        elseif( -not $env:AppVeyor)
-        {
-            <#
-                On Windows, but not an AppVeyor and pre-requisites are missing
-                For now we will skip, and write a warning.  Work to resolve this is tracked in:
-                https://github.com/PowerShell/PowerShell/issues/3429
-            #>
-            Write-Warning "Node and npm are required to run this test"
-            $skip = $true
-        }
+        <#
+            On Windows, pre-requisites are missing
+            For now we will skip, and write a warning.  Work to resolve this is tracked in:
+            https://github.com/PowerShell/PowerShell/issues/3429
+        #>
+        Write-Warning "Node and npm are required to run this test"
+        $skip = $true
```

Comment:
Oh, It seems logic above was broken and is still broken.

Reply:
What do you mean?

Reply:
We need if-else.

Reply:
Ah i see.   Is it just 
if ( $IsWindows ) {} 
?

Reply:
```powershell
    BeforeAll {

        Push-Location $psscriptroot
        $skip = $false
        $NpmInstalled = "not installed"
        if (Get-Command -Name 'npm' -ErrorAction SilentlyContinue)
        {
            $NpmInstalled = "Installed"
            Write-Verbose -Message "NPM is checking Gulp is installed. This may take a few moments." -Verbose
            start-nativeExecution { npm install --silent }
            start-nativeExecution { npm install 'gulp@4.0.0' --silent }
            if(!(Get-Command -Name 'gulp' -ErrorAction SilentlyContinue))
            {
                start-nativeExecution {
                    sudo npm install -g 'gulp@4.0.0' --silent
                    # Sometimes this folder is left behind with root permissions and is needed by later NPM installs which don't need sudo
                    sudo rm -rf ~/.npm/_cacache
                }
            }
        }

        if(!(Get-Command -Name 'node' -ErrorAction SilentlyContinue))
        {
            <#
                On Windows, pre-requisites are missing
                For now we will skip, and write a warning.  Work to resolve this is tracked in:
                https://github.com/PowerShell/PowerShell/issues/3429
            #>
            Write-Warning "Node and npm are required to run this test"
            $skip = $tru
        }
```

Reply:
Thanks.  I'll work on putting this in.

Reply:
should be added now 👍